### DISCUSSION TITLE
SWC - Add support for externalHelpers

### DISCUSSION THREAD

**Discussion by secundant** [vercel/next.js#31913] [Ideas]
### Describe the feature you'd like to request

By default swc generates new helper utility for every usage in source code. This leads to a continuous increase of application bundle by duplicating code without any significant purpose.
![spread-1](https://user-images.githubusercontent.com/31571122/143719272-b1182d2b-ea42-4a9b-ae9c-b2d5802c7e64.png)
![spread-2](https://user-images.githubusercontent.com/31571122/143719273-5b019b6a-2fb2-4762-b43e-9855be6975ca.png)
![spread-3](https://user-images.githubusercontent.com/31571122/143719274-a8335617-c035-4957-ae4a-bc4018414bce.png)

### Describe the solution you'd like

Add @swc/helpers internally and enable "jsc.externalHelpers" (https://swc.rs/docs/configuration/compilation#jscexternalhelpers) by default

### Describe alternatives you've considered

N/A

**Comment by timneutkens** [comment]
There's an open issue for this: #30806
